packages/textkit/src/engines/linebreaker/index.js

```diff
@@ -137,7 +137,10 @@ const getNodes = (attributedString, { align }, options) => {
 
       acc.push(linebreak.glue(width, value, stretch, shrink));
     } else {
-      const hyphenated = !hasOnlySpaces(syllables[index + 1]);
+      const hyphenated =
+        options.hyphenationCallback === null
```

Comment:
@SeniyaSansith can we add a comment here to describe the logic used here?
Someone looking through the code might not understand.
i.e. what does it mean for the `hyphenationCallback` to be null?